### PR TITLE
Improve accessibility

### LINKED ISSUE #175: Improve accessibility
According to W3 ARIA, followings issues should be considered: 

- [x] Burger button does not have an accessible name.
- [x] Links to Twitter and Github in menu don't have discernable names.

- [x] In the burger button `div` is not allowed as child of `button` in this context 

- [x] `<html>` element does not have a `[lang]` attribute.

- [x] `frameborder` attribute for `iframe` tag is obsolete, use CSS instead.
- [x] `scrolling` attribute for `iframe` tag is obsolete, use CSS instead. 

- [x] Stray start tag `script`.

### PR DESCRIPTION
**Pull request by justinezahiri**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no 
| New feature?  | yes/no 
| BC breaks?    | no
| Deprecations? | yes/no
| Tickets       | fixes #175  
| License       | MIT
| Doc PR        | api-platform/docs#...

Fixes some issues related to accessibility, according to W3 validator, on homepage of api-platform website.

### CODE REVIEW

File: src/styles/components/_footer.scss [hunk 1]
```diff
@@ -38,12 +38,22 @@
 
   .footer__follow {
     margin-top: 10px;
+
```
Comment (alanpoulain) [inline src/styles/components/_footer.scss]: To remove.

File: src/styles/components/_header.scss [hunk 2]
```diff
@@ -186,19 +186,22 @@
...
+
   @include mq($max-width: $vw-huge) {
-    .menu-item > .menu-item__link {
+    .menu-item>.menu-item__link {
       padding: 0 10px;
     }
   }
+
   @include mq($max-width: $vw-mobile-menu) {
+
```
Comment (alanpoulain) [inline src/styles/components/_header.scss]: To remove.

File: src/components/layout/Header.js [hunk 3]
```diff
@@ -17,10 +17,10 @@ const Header = () => (
     <Search className="header__search" />
     <nav className="header__nav">{nav}</nav>
     <nav className="header__social">
-      <a href="https://twitter.com/ApiPlatform" target="blank">
+      <a href="https://twitter.com/ApiPlatform" target="blank" aria-label="find us on twitter">
```
Comment (dunglas) [inline src/components/layout/Header.js]: should be `_blank`, not `blank` right? And so we must add `rel="noopener noreferrer"`
Same for other similar links

File: src/styles/components/_footer.scss (hunk 1 above)
Comment (dunglas) [inline src/styles/components/_footer.scss]: Should be reverted

**Comment by dunglas** [comment]
Thanks Justine!

**Pull request merged**